docs/development/workflow/development_workflow.rst

```diff
@@ -584,10 +579,14 @@ Behind the scenes, `git`_ is deleting the changes and branch you made, making th
 changes others made to the development branch of Astropy, then re-making your
 branch from the development branch and applying your changes to your branch.
 
-The actual rebasing is usually easy::
+First, fetch the latest development astropy and go to your branch of interest::
+
+    git fetch astropy main
```

Comment:
Maybe just `git fetch astropy`? That fetches everything that is new, but I think that is good practice anyway...

p.s. The reorganization of checking out one's own branch is much nicer!